nixos/tests/ente/snakeoil-certs.nix

```diff
@@ -0,0 +1,14 @@
+let
```

Comment:
Why are you re-creating the same logic as `common/acme/server/snakeoil-certs.nix` which is used in other NixOS tests?

Reply:
Because it only issues a certificate for acme.test, but we need a certificate for several subdomains.